README.md

```diff
@@ -103,3 +100,64 @@ $ ./setup <app name> <plan>
 $ heroku kafka:topics:write [prefix]textlines "hello world" -a <app>
 $ heroku pg:psql -c 'select * from windowed_counts' HEROKU_POSTGRESQL_URL -a <app>
 ```
+
+### Example Use Cases
+
+Now let's use Kafka Streams with some example use cases. The `data-generators` directory contains some simple Ruby scripts to generate streams of data. Instructions on how to use them are below.
+
+#### Word Count
+
+First we'll do word count over a large stream of text. This will produce into Kafka lines from _Alice's Adventures in Wonderland_.
+
+```bash
+$ heroku run ruby data-generators/text-generator/stream-lines-to-kafka.rb data-generators/text-generator/alice-in-wonderland.txt --app sushi
+```
+
+Alternatively, if you have Ruby and Bundler installed locally, you can run the data generator locally
+
+```bash
+$ bundle install --path=vendor/gems
+$ cd data-generators/text-generator
+$ HEROKU_KAFKA_URL=$(heroku config:get HEROKU_KAFKA_URL) \
+HEROKU_KAFKA_CLIENT_CERT=$(heroku config:get HEROKU_KAFKA_CLIENT_CERT) \
+HEROKU_KAFKA_CLIENT_CERT_KEY=$(heroku config:get HEROKU_KAFKA_CLIENT_CERT_KEY) \
+HEROKU_KAFKA_TRUSTED_CERT=$(heroku config:get HEROKU_KAFKA_TRUSTED_CERT) \
+HEROKU_KAFKA_PREFIX=$(heroku config:get HEROKU_KAFKA_PREFIX) \
+bundle exec ruby stream-lines-to-kafka.rb alice-in-wonderland.txt
+```
+
+Now we can see the word count for specific time windows:
+
+```bash
+$ heroku pg:psql -c 'select * from windowed_counts order by count desc' HEROKU_POSTGRESQL_URL
```

Comment:
We should order this by `time_window`. This would allow you to get a sense of where words appear in the text.